Public/Src/Tools/Execution.Analyzer/Analyzers.Core/XLGPlusPlus/XLGppDataStore.cs

```diff
@@ -0,0 +1,115 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using BuildXL.Engine.Cache.KeyValueStores;
+using BuildXL.Execution.Analyzer;
+using BuildXL.Scheduler.Tracing;
+using BuildXL.Utilities;
+using Google.Protobuf;
+
+namespace BuildXL.Analyzers.Core.XLGPlusPlus
+{
+    public class XLGppDataStore
+    {
+        /// <summary>
+        /// Rocks DB Accessor for XLG++ data
+        /// </summary>
+        public KeyValueStoreAccessor Accessor { get; set; }
+
+        public XLGppDataStore()
+        {
+
+        }
+
+        /// <summary>
+        /// Open the datastore and populate the KeyValueStoreAccessor for the XLG++ DB
+        /// </summary>
+        /// <param name="storeDirectory"></param>
+        /// <param name="defaultColumnKeyTracked"></param>
+        /// <param name="additionalColumns"></param>

```

Comment:
We have a convention of ommiting the param documentation if there is nothing useful mentioned.

Reply:
These generated by default when I just typed ///, I'll keep that in mind for next time. 

---
In reply to: [305539058](https://github.com/microsoft/BuildXL/pull/636#discussion_r305539058) [](ancestors = 305539058)